### PR TITLE
Mouse speed tab

### PR DESCRIPTION
**Pull request by NejcZdovc**
PR based on #49

### CODE REVIEW

**Comment by NejcZdovc** [comment]
@mondalaci What do you think?

**Comment by mondalaci** [comment]
@NejcZdovc Looks very nice! I can think of the following changes:

* Move the "You can set the multiplier in the setting." text below the buttons and change the "the setting" part to "settings".
* It'd be worth to put `cursor:pointer` to the side menu elements: Move, Scroll, Click, Speed

**Comment by NejcZdovc** [comment]
@mondalaci Done.

**Comment by fjozsef** [comment]
@mondalaci Should the 'the' word part of the hyperlink or not? I think it shouldn't. I would also suggest the title to be 'Settings' with capital S.
mouse

**Comment by mondalaci** [comment]
@fjozsef Agreed!

**Comment by NejcZdovc** [comment]
Fixed text

File: src/components/popover/tab/mouse/mouse-tab.component.html [hunk 1]
```diff
@@ -84,15 +84,25 @@
...
                 [class.btn-primary]="mouseActionParam === MouseActionParam.accelerate"
-                (click)="setMouseActionParam(MouseActionParam.accelerate)">+</button>
+                (click)="setMouseActionParam(MouseActionParam.accelerate)"
+            >
+                +
+                <span>Accelerate</span>
+            </div>
+        </div>
+        <div class="help-text--mouse-speed last-help  text-left">
+            <p>You can set the multiplier in the <a [routerLink]="['/settings']" title="settings">Settings</a>.</p>
```
Comment (fjozsef) [inline src/components/popover/tab/mouse/mouse-tab.component.html]: This should be the fix I mentioned: `title="Settings">settings`
Title with capital S, but the text is lower s.

Comment (NejcZdovc) [inline src/components/popover/tab/mouse/mouse-tab.component.html]: Done

**Pull request merged**